src/checkbox.ts

```diff
@@ -177,6 +174,22 @@ export default class Checkbox extends LitElement implements FormControl {
     this.#setValidity();
   }
 
+  /**
+   * @default undefined
+   */
+  @property()
+  get split(): 'left' | 'middle' | 'right' | undefined {
+    return this.#split;
+  }
+
+  set split(split: 'left' | 'middle' | 'right' | undefined) {
+    if (this.orientation === 'vertical') {
```

Comment:
Would this be overkill?

```suggestion
    if (split && this.orientation === 'vertical') {
```

In case somehow this `split` setter validly hits with it being set to `undefined`

Reply:
Not to me. That seems right given consumers can split and unsplit. I'll ignore this suggestion and push a commit changing it everywhere.

Reply:
https://github.com/CrowdStrike/glide-core/pull/1199/commits/87233391570e02ea728ad7701dfd95a82d703088 ✅